index.css

```diff
@@ -1,6 +1,100 @@
-header{
-    background-color: #3C3A39;
+.content{
+background: #3C3A39;
+ width: Hug(375)px;
+height: Hug(44)px;
+gap: 0px;
+opacity: 0px;
+
```

Comment:
Consider not to leave an empty space inside your selector, kindly remove all of them